### PR TITLE
prepare GA for azure-core-management

### PR DESCRIPTION
**Pull request by weidongxu-microsoft**
Version 1.0.0

### CODE REVIEW

**Review by ChenTanyi — APPROVED** [review]

**Review by JimSuplizio — CHANGES_REQUESTED** [review]
The changes look good for the pom files and changelogs but you do need to update the versions in the following readme files as well

- [sdk/core/azure-core-management/README.md](https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/core/azure-core-management/README.md)
- [sdk/resourcemanager/azure-resourcemanager-resources/README.md](https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/resourcemanager/azure-resourcemanager-resources/README.md)
- [sdk/resourcemanager/azure-resourcemanager-test/README.md](https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/resourcemanager/azure-resourcemanager-test/README.md)

Once these updates to the README files are made, please do not wait for _my_ approval as I'm currently on leave.

**Comment by weidongxu-microsoft** [comment]
@JimSuplizio  Thanks for the notification. Sure, I will run the script again without `--skip-readme`.

**Comment by weidongxu-microsoft** [comment]
sdk/core/azure-core-management/README.md updated.

File: sdk/core/azure-core-management/CHANGELOG.md
Comment (samvaity) [inline sdk/core/azure-core-management/CHANGELOG.md:5]: Do consider breaking the updates into following sections https://azure.github.io/azure-sdk/policies_releasenotes.html#whats-a-developer-impacting-change

Comment (weidongxu-microsoft) [reply sdk/core/azure-core-management/CHANGELOG.md:5]: Thanks for the info. Get the idea, minor bug fix does not need get into CHANGELOG, and major interface change and feature support will need be there in CHANGELOG.

**Pull request merged**
